### PR TITLE
Use `type-fest`

### PR DESCRIPTION
**Pull request by CvX**
🚀

### CODE REVIEW

File: index.test-d.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ import {
 } from '.';
 
 expectType<AllElectron | Remote>(api);
-expectType<boolean>(api.app.isPackaged);
+expectType<boolean | undefined>(api.app.isPackaged);
```
Comment (sindresorhus) [inline index.test-d.ts]: Why is this `undefined`?

Comment (CvX) [reply index.test-d.ts]: In [electron-api.json](https://github.com/electron/electron/releases/download/v4.1.0/electron-api.json) `isPackaged` property isn't marked as `required`, so in the [electron/typescript-definitions](https://github.com/electron/typescript-definitions) generated electron.d.ts file it is added as:

```ts
interface App extends EventEmitter {
  (…)
  isPackaged?: boolean;
}
```

hence `boolean | undefined`.

Comment (sindresorhus) [reply index.test-d.ts]: Ugh, the Electron and Node.js type definitions are just so bad...

**Comment by sindresorhus** [comment]
Thanks for doing this :)

**Pull request merged**